docs/RunningFirstBenchmark.md

```diff
@@ -0,0 +1,114 @@
+# Running Your First Benchmark
+
+This tutorial is aimed at people new to the library and shows you how to run benchmarks with the Google Benchmark library.
+
+# Install the prerequisites
+
+We need the following programs:
+
+* git : to clone google benchmark and gtest libraries
+* cmake : to build google benchmark
+* make : to build google benchmark. Advanced users can use an alternative cmake backend like ninja.
+* g++ : to compile your c++ code. Advanced users can use other compilers like clang instead.
+
+On Fedora 28, the following command installs the requirements:
+
+```
+sudo dnf install --assumeyes wget git-core cmake make gcc-c++
+```
+
+At the time of writing, the versions were as follows:
+* g++ : 8.1.1 20180502
+* cmake : 3.11.2
+* gnu make: 4.2.1
+* git: 2.17.1
+
+# Build and install Google Benchmark
+
+Follow the instructions elsewhere to build and install the google-benchmark library.
+
+On Fedora 28, these commands were sufficient:
+
+```
+git clone https://github.com/google/benchmark.git
+cd benchmark
+git clone https://github.com/google/googletest.git
+mkdir build
+cd build
+cmake .. -DCMAKE_BUILD_TYPE=RELEASE
+make -j
+sudo make install
+```
+
+`make install` should result in lines that look like:
+
+```
+-- Install configuration: "RELEASE"
```

Comment:
This is quite messy.
I'm thinking that maybe you either want to specify a custom install prefix, or not install at all.
In either case, you'll need to specify the include/link paths.

Or maybe just use the packaged version?

Reply:
Sadly, google benchmark is not available in the Fedora 28 repository right now. 

I'll prepare instructions for the no install use case. The "install" related bits can then be an "appendix" in this document or just removed.